koji_containerbuild/cli.py

```diff
@@ -103,7 +103,16 @@ def handle_container_build(options, session, args):
         parser.error(_("Exactly two arguments (a build target and a SCM URL "
                        "or archive file) are required"))
         assert False
-    clikoji.activate_session(session)
+    # Koji API has changed - activate_session requires two arguments
+    try:
+        clikoji.activate_session(session)
+    except TypeError:
+        clikoji.activate_session(session, options)
```

Comment:
Looks like `activate_session` can now also be imported from `koji_cli.lib`. Let's use that here, too.
We can also perform this check earlier on, something like:

```python

clikoji = None

try:
  from koji_cli.lib import _running_in_bg, activate_session
except ImportError:
  # Create wrappers for backwards compatibility.
  _running_in_bg = clikoij._runing_in_bg

  def activate_session(session, options):
    try:
      clikoji.activate_session(session)
    except TypeError:
      clikoji.activate_session(session, options)
  
```